src/c++/perf_analyzer/main.cc

```diff
@@ -557,6 +558,20 @@ Usage(char** argv, const std::string& msg = std::string())
                    "sequence will be within +/- 20% of the base length.",
                    18)
             << std::endl;
+  std::cerr
+      << FormatMessage(
+             " --sequence-id-range <start:end>: Determines the range of "
+             "sequence id used by the perf_analyzer. The perf_analyzer "
+             "will start from the sequence id of 'start' and go till "
+             "'end' (excluded). If 'end' is not specified then perf_analyzer "
+             "will "
+             "use new sequence id without bounds. If 'end' is specified and "
+             "the "
+             "concurrency setting may result in maintaining a number of "
+             "sequences more than the range of available sequence id, a "
```

Comment:
Can we instead make the perf_analyzer fail with an error in the case where the range of sequence ids is not sufficient for the number of sequences that the perf_analyzer is trying to create. Also mention what the default is.